subsys/bluetooth/controller/ll_sw/nordic/lll/lll_adv_iso.c

```diff
@@ -443,7 +449,27 @@ static int prepare_cb_common(struct lll_prepare_param *p)
 	LL_ASSERT(!ret);
 
 	/* Calculate ahead the next subevent channel index */
-	next_chan_calc(lll, event_counter, data_chan_id);
+	if (false) {
+
+#if defined(CONFIG_BT_CTLR_ADV_ISO_SEQUENTIAL)
+	} else if (lll->bis_spacing >= (lll->sub_interval * lll->nse)) {
+		next_chan_calc_seq(lll, event_counter, data_chan_id);
+#endif /* CONFIG_BT_CTLR_ADV_ISO_SEQUENTIAL */
+
+#if defined(CONFIG_BT_CTLR_ADV_ISO_INTERLEAVED)
+	} else if (lll->bis_spacing < (lll->sub_interval * lll->nse)) {
```

Comment:
You may want to put `lll->bis_spacing < (lll->sub_interval * lll->nse` into a local variable (if possible)